packages/react-components/react-button/src/components/MenuButton/useMenuButton.tsx

```diff
@@ -26,6 +27,10 @@ export const useMenuButton_unstable = (
       menuIcon: 'span',
     },
 
+    root: {
+      ...buttonState.root,
+      'aria-expanded': props['aria-expanded'] ?? false,
+    },
```

Comment:
Could you assign `buttonState.root['aria-expanded'] ??= false` above, instead of doing a spread here?